components/brave_wallet_ui/utils/pricing-utils.ts

```diff
@@ -0,0 +1,32 @@
+// Copyright (c) 2022 The Brave Authors. All rights reserved.
```

Comment:
Should we add a unit test for this?

Reply:
Added here: [test: pricing-utils](https://github.com/brave/brave-core/pull/14818/commits/c24dc2b4d124d8d484eae30e193d972ec558d01d)